io-package.json

```diff
@@ -508,6 +508,24 @@
                 "alarm_code": 1234
             }
         },
+        {
+            "_id": "control.alarm.arm_state",
```

Comment:
It is not strictly forbidden to use other state-objects as parent, but also not allowed and from what I hear not liked much. So please change id in something like control.alarm_arm_state or similar. So that it will show next to control.alarm but not as child of it.